### ISSUE TITLE
how to kill or close lua runtime?

### ISSUE THREAD

**Issue by user20272912** [scoder/lupa#111]
can kill or close lua runtime?

lua = LuaRuntime()

lua.execute('while true do print(\'loop run~~~~\') end')

lua.kill() < -- how this?

**Comment by scoder** [comment]
No, such functionality is not available. There might be a way to do that in Lua, though.

**Comment by user20272912** [comment]
how can i do in Lua?

**Comment by scoder** [comment]
You should better ask that in a Lua forum or mailing list.

However, the first question I would ask is: are you sure that you need to do that? Maybe there is a better solution to the actual problem that you have? (You didn't say what that problem actually is that you are trying to solve.)
